src/Sentry/Internal/MainExceptionProcessor.cs

```diff
@@ -110,14 +110,24 @@ internal static Mechanism GetMechanism(Exception exception)
         {
             Debug.Assert(exception != null);
 
-            Mechanism mechanism = null;
+            Mechanism mechanism = new Mechanism()
+            {
+                Handled = true
```

Comment:
Can we leave this `null` by default instead of defaulting to `true`?
Not sure if Sentry expects it to be explicitly set to `true` though

Reply:
![image](https://user-images.githubusercontent.com/8229322/77917300-c974cc80-7270-11ea-8bee-9720b52a69cc.png)
The handled inside of Mechanism doesn't seems to affect that "handled vs unhandled", it only point that the exception was handled or not.
That's why I was thinking that by default, an exception is handled, unless it comes from the Unhandled Exception Integration.